ehrql/dummy_data_nextgen/generator.py

```diff
@@ -4,6 +4,7 @@
 import random
 import string
 import time
+from bisect import bisect_left
```

Comment:
The contents of Python's stdlib are endlessly surprising to me! Nice to add another one to the bag of "things I will no doubt find a use for now I know they exist".

Reply:
I must admit about 80% of the time I find it easier to write my own binary search and the remaining 20% of the time Claude helps me get the fencepost conditions right.